src/Component/GeneralEntity/GeneralEntityTable/GeneralEntityTable.tsx

```diff
@@ -110,15 +110,14 @@ export function GeneralEntityTable<T extends BaseEntity>({
   const onRowClick = (record: T) => navigate(`${routePath}/${record.id}`);
 
   const onDeleteClick = async (record: T) => {
-
     const entityId = record?.id;
 
     if (!entityId) {
       fetchEntities();
       return;
     }
 
-    let entityName = (record as any).name || (record as any).title;
+    let entityName: string | number = (record as any).name || (record as any).title;
```

Comment:
Minor: Since this is names and titles, I would suggest using only string as type and cast all the numbers as string. For me it makes more sense that names and titles are always strings. Or is there a reason for it to also be a number?

Reply:
We guess that the entity name is set in the `name` or `title` attribute of the entity. If this is not available (e.g. for imagefiles), we fall back to the id of the entity.